File: app/src/main/java/ru/github/pvtitov/senatapp/pojos/Protocol.java
package ru.github.pvtitov.senatapp.pojos;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Protocol {

    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("category")
    @Expose
    private String category;
    @SerializedName("currentVersion")
    @Expose
    private CurrentVersion currentVersion;
    @SerializedName("id")
    @Expose
    private String id;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getCategory() {
        return category;
    }

    public void setCategory(String category) {
        this.category = category;
    }

    public CurrentVersion getCurrentVersion() {
        return currentVersion;
    }

    public void setCurrentVersion(CurrentVersion currentVersion) {
        this.currentVersion = currentVersion;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

}

File: app/src/main/java/ru/github/pvtitov/senatapp/pojos/Protocol.kt
package ru.github.pvtitov.senatapp.pojos

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Protocol {

    @SerializedName("name")
    @Expose
    var name: String = ""
    @SerializedName("category")
    @Expose
    var category: String = ""
    @SerializedName("currentVersion")
    @Expose
    var currentVersion: CurrentVersion? = null
    @SerializedName("id")
    @Expose
    var id: String = ""

}